src/platforms/react-native/manual-setup/codepush.mdx

```diff
@@ -46,6 +46,12 @@ codePush.getUpdateMetadata().then(update => {
 });
 ```
 
+If you're wrapping your root component with `Sentry.wrap` to access Performance features, `codePush` should be the outer most function because it needs access to the very root component in order to swap out the bundle.
```

Comment:
```suggestion
If you're wrapping your root component with `Sentry.wrap` to access Performance features, `codePush` should be the outer most function because it needs access to the root component in order to swap out the bundle.
```